app/pages/projects.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Head from 'next/head';
+
+import { useRouter } from 'next/router';
+import { useAuth } from 'hooks/authentication';
+
+import Header from 'layout/header';
+import Projects from 'layout/projects';
+import Protected from 'layout/protected';
+
+const ProjectsPage: React.FC = () => {
+  const { user, errorRedirect } = useAuth();
+  const router = useRouter();
+
+  if (!user) {
+    router.push(errorRedirect);
+    return null;
+  }
```

Comment:
This is already handled by the `Protected` component.

Reply:
Thanks for the catch